### PR TITLE
#1239: Helm v9.2.0 Release

### PR DESCRIPTION
**Pull request by oderayi**
- Updated configurations and environment variables
- Updated all services to latest published image versions

### CODE REVIEW

File: bulk-api-adapter/chart-service/Chart.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: bulk-api-adapter API component Helm chart for Kubernetes
 name: bulk-api-adapter-service
-version: 8.8.0
-appVersion: "8.8.0-snapshot"
+version: 9.2.0
+appVersion: "9.1.0-snapshot"
```
Comment (rmothilal) [inline bulk-api-adapter/chart-service/Chart.yaml]: this needs to be 9.2.0-snapshot

File: bulk-api-adapter/chart-handler-notification/Chart.yaml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: bulk-api-adapter Handler for Notifications component Helm chart for Kubernetes
 name: bulk-api-adapter-handler-notification
-version: 8.8.0
-appVersion: "8.8.0-snapshot"
+version: 9.2.0
+appVersion: "9.1.0-snapshot"
```
Comment (rmothilal) [inline bulk-api-adapter/chart-handler-notification/Chart.yaml]: this needs to be 9.2.0-snapshot

File: account-lookup-service/chart-service/Chart.yaml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-version: 8.8.0
-appVersion: "9.2.0"
+version: 9.2.0
+appVersion: "9.2.1"
```
Comment (rmothilal) [inline account-lookup-service/chart-service/Chart.yaml]: this needs to be 9.2.2

File: bulk-api-adapter/chart-service/Chart.yaml (hunk 1 above)
Comment (elnyry) [reply bulk-api-adapter/chart-service/Chart.yaml]:
```suggestion
appVersion: "9.2.0-snapshot"
```

**Pull request closed without merging**